app/views/partials/paymentAllocations/PaymentAllocationsCreditAmount.scala.html

```diff
@@ -35,21 +35,11 @@
 
 @getLink = @{
     if (creditsRefundsRepayEnabled) {
-        if(isAgent) {
```

Comment:
Now that the there's only the `moneyOnAccount` message, the check if the user is an agent is no longer required. The message is the same regardless of user type.